SECURITY.md

```diff
@@ -0,0 +1,175 @@
+  fixed, according to the Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities
+  which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of
+  data you access to the minimum required for effectively demonstrating a Proof
+  of Concept; and cease testing and submit a report immediately if you encounter
+  any user data during testing, such as Personally Identifiable Information
+  (PII), Personal Healthcare Information (PHI), credit card data, or proprietary
+  information;
+- You should only interact with test accounts and Knative instances you own or
+  with explicit permission from the account holder; and
+- Do not engage in extortion.
+
+# Safe Harbor
+
+When conducting vulnerability research according to this policy, we consider
+this research to be:
+
+- Authorized in accordance with the Computer Fraud and Abuse Act (CFAA) (and/or
+  similar state laws), and we will not initiate or support legal action against
+  you for accidental, good faith violations of this policy;
+- Exempt from the Digital Millennium Copyright Act (DMCA), and we will not bring
+  a claim against you for circumvention of technology controls;
+- Exempt from restrictions in our Terms & Conditions that would interfere with
+  conducting security research, and we waive those restrictions on a limited
+  basis for work done under this policy; and
+- Lawful, helpful to the overall security of the Internet, and conducted in good
+  faith.
+
+You are expected, as always, to comply with all applicable laws.
+
+If at any time you have concerns or are uncertain whether your security research
+is consistent with this policy, please submit a report through one of our
+[Official Channels](#reporting-a-vulnerability) before going any further.
+
+# Scope and Supported Versions
+
+We (the community) support the 4 most recent minor versions of Knative. For more
+details on what this means, see our
+[release principles](https://knative.dev/community/contributing/mechanics/release-versioning-principles/).
+
+Repos with releases have the releases listed in the Github UI:
+
+TODO: move this list elsewhere?
+
+- [serving](https://github.com/knative/serving/releases)
+- [eventing](https://github.com/knative/eventing/releases)
+- [client](https://github.com/knative/client/releases)
+- [operator](https://github.com/knative/operator/releases)
+- [eventing-contrib](https://github.com/knative/eventing-contrib/releases)
+- [docs](https://github.com/knative/docs/releases)
+
+Repos without releases are supported only on the default branch.
+
+TODO: Alternative: Vulnerabilities in repos without releases are in scope only
+if they can be exploited in a release.
+
+TODO: any other repos with releases? TODO: what about knative-sandbox?
+
+No support is provided for archived repos.
+
+# Reporting a Vulnerability
+
+Please email the
+[Knative Security Committee](mailto:knative-security@googlegroups.com) with
```

Comment:
I'd replace "Committee" with "list", IIRC this is steering (which I find odd as this seems technical, but 🤷 ).